flink-ml-benchmark/src/main/java/org/apache/flink/ml/benchmark/datagenerator/clustering/KMeansModelDataGenerator.java

```diff
@@ -55,20 +61,36 @@ public Table[] getData(StreamTableEnvironment tEnv) {
         InputDataGenerator<?> vectorArrayGenerator = new DenseVectorArrayGenerator();
         ReadWriteUtils.updateExistingParams(vectorArrayGenerator, paramMap);
         vectorArrayGenerator.setNumValues(1);
+        vectorArrayGenerator.setColNames(new String[] {"centroids"});
```

Comment:
Why do we need to make this change? This is change necessary to add benchmarks for algorithms targeted by this PR?

Reply:
Yes, it is necessary. Because there's a mistake in `KmeansModelDataGenerator` which would break the test for `benchmark-demo.json`.

Reply:
Adding the first line `vectorArrayGenerator.setColNames(new String[] {"centroids"});` is because `DenseVectorArrayGenerator` requires `columnNames` is set, otherwise NPE would be thrown. 
The other changes are to reduce the operation of `toDataStream` and `fromDataStream`, just do the conversion base on `Table`.

Reply:
Sounds good. Thanks for the explanation.